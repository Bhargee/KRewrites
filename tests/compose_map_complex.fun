"map" (fun a -> a + 1) "map" (fun b -> b * 3) "map" (fun c -> c / 3)
