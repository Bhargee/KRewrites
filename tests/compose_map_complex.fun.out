<T>
    <k>
        "map" (fun (#symExp(2) -> ((fun (#symExp(1) -> ((fun (a -> (a + 1
          ))) ((fun (b -> (b * 3))) #symExp(1))))) ((fun (c -> (c / 3))) 
          #symExp(2)))))
    </k>
    <store>
        .Map
    </store>
    <env>
        .Map
    </env>
</T>
